instrumentation/opentelemetry-instrumentation-httpx/src/opentelemetry/instrumentation/httpx/__init__.py

```diff
@@ -31,7 +31,7 @@
 from opentelemetry.trace.span import Span
 from opentelemetry.trace.status import Status
 
-URL = typing.Tuple[bytes, bytes, typing.Optional[int], bytes]
+RawURL = typing.Tuple[bytes, bytes, typing.Optional[int], bytes]
```

Comment:
Worth adding a comment documenting this does not contain credentials or other additional info?

Reply:
Hm, I agree with @owais. "Raw" does not significantly tells the reader that the URL has no credentials. I'd say the we just keep the `URL` name and add the tests below.